index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang = "en">
+ <head>
+   <meta charset = "UTF-8">
+   <title> </title>
+   </head>
+   <body>
+     <form action = "/ha" method = "GET" name = "getRequest">
```

Comment:
method is unnecessary, check it out yo: https://github.com/seemcat/garage_script/pull/7/files